### ISSUE TITLE
See the automatic configuration values

### ISSUE THREAD

**Issue by dht** [wallabyjs/public#2641]
In VSCode I choose `Automatic Configuration <Project directory>`.
How can I see the values of the auto generated automatic configuration?
I want to copy it to the `wallaby.js` file and tweak it.

**Comment by smcenlly** [comment]
You can see the results of the automatic configuration in your [Wallaby Diagnostics Report](https://wallabyjs.com/docs/intro/troubleshooting.html#diagnostics-report) but taking this configuration and copying it/modifying in a `wallaby.js` file is unlikely to work for you and there are better ways of modifying the configuration.

Can you please describe what you want to tweak? If you are using automatic configuration, then Wallaby will configure and run your tests the same way as they run outside of Wallaby (e.g. with CLI) and in most cases you will want to change your test runner configuration outside of Wallaby to work the same way that you want Wallaby to.

If you do need to tweak some settings, it is better to [override automatic defaults](https://wallabyjs.com/docs/intro/config.html#overriding-automatic-defaults) vs. creating a brand new configuration. If you are using Jest, there is more detail on overriding jest automatic configuration defaults in our [jest docs](https://wallabyjs.com/docs/integration/jest.html#overriding-automatic-configuration). If you are using Angular, there is more detail about overriding our automatic configuration settings in our [angular docs](https://wallabyjs.com/docs/integration/angular.html#automatic-configuration-workspace-overrides).

**Comment by smcenlly** [comment]
Closing this issue as we have not had a response in 7 days.

**Comment by dht** [comment]
My root issue was failing to setup wallaby on a specific mono-repo package. I though seeing the automatic configuration would lead me to the right direction. However, you are right, those configurations are cumbersome though interesting to investigate for curiosity sake.

As far as my root issue, I managed to find a solution: I've installed `ts-jest`.
The clue that lead me to the solution was simple: I just ran jest from the CLI and realized the error wallaby was showing me was a jest error. After installing `ts-jest` and initializing it's config, both jest and wallaby are running smoothly.

Thanks Simon for a good product and a good support service. It must be challenging with the time differences.
